manual/vcredist2008/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,15 @@
+﻿$packageName = 'vcredist2005'
+$packageSearch = "Microsoft Visual C++ 2008 Redistributable*"
```

Comment:
I know, I'm just thinking that *PERHAPS* it will fix the strange uninstall behavior you mentioned.